models/iaf_psc_exp.h

```diff
@@ -93,13 +78,20 @@ matrix objects.
   For implementation details see the
   `IAF_neurons_singularity <../model_details/IAF_neurons_singularity.ipynb>`_ notebook.
 
-iaf_psc_exp can handle current input in two ways: Current input
-through receptor_type 0 are handled as stepwise constant current
-input as in other iaf models, i.e., this current directly enters
-the membrane potential equation. Current input through
-receptor_type 1, in contrast, is filtered through an exponential
-kernel with the time constant of the excitatory synapse,
-tau_syn_ex. For an example application, see [4]_.
+iaf_psc_exp can handle current input in two ways:
+
+1. Current input through receptor_type 0 are handled as stepwise constant
+   current input as in other iaf models, i.e., this current directly enters the
+   membrane potential equation.
+2. Current input through receptor_type 1, in contrast, is filtered through an
+   exponential kernel with the time constant of the excitatory synapse,
+   ``tau_syn_ex``.
+
+   For an example application, see [4]_.
+
+   **Warning:** this current contribution is added to the state variable
+   ``i_syn_ex_``, which changes its numerical values in case it is being
```

Comment:
This sentence is confusing. 
What  numerical values change? Can you be explicit with "being recorded" (what does the actual syntax look like).